src/java.base/share/classes/jdk/internal/util/ArraysSupport.java

```diff
@@ -575,52 +575,83 @@ public static int mismatch(double[] a, int aFromIndex,
     }
 
     /**
-     * The maximum length of array to allocate (unless necessary).
-     * Some VMs reserve some header words in an array.
-     * Attempts to allocate larger arrays may result in
-     * {@code OutOfMemoryError: Requested array size exceeds VM limit}
+     * A soft maximum array length imposed by array growth computations.
+     * Some JVMs (such as Hotspot) have an implementation limit that will cause
```

Comment:
The numbers of spellings "HotSpot" to that of "Hotspot" in the JDK codebase is 10 to 1 respectively. Also, on my machine I can see this:

```
$ java --version
java 15 2020-09-15
Java(TM) SE Runtime Environment (build 15+36-1562)
Java HotSpot(TM) 64-Bit Server VM (build 15+36-1562, mixed mode, sharing)
```

Reply:
Fixed.